### PR TITLE
修复同步keep数据出现的错误

### PR DESCRIPTION
**Pull request by abinnz**

### CODE REVIEW

File: src/utils/utils.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const scrollToMap = () => {
 const locationForRun = (run) => {
   const location = run.location_country;
   let [city, province, country] = ['', '', ''];
-  if (location) {
+  if (location && location != "None") {
```
Comment (yihong0618) [inline src/utils/utils.js:37]: I suggest add `if` here,  and we do not need to compare with the location string
```javascript
    const countryMatch = l[l.length - 1].match(/[\u4e00-\u9fa5].*[\u4e00-\u9fa5]/) || l[2].match(/[\u4e00-\u9fa5].*[\u4e00-\u9fa5]/);
```

File: scripts/keep_sync.py [hunk 2]
```diff
@@ -171,7 +171,7 @@ def parse_points_to_gpx(run_points_data, start_time):
             {
                 "latitude": point["latitude"],
                 "longitude": point["longitude"],
-                "elevation": point["verticalAccuracy"],
+                "elevation": point["verticalAccuracy"] if "verticalAccuracy" in point.keys() else None,
```
Comment (yihong0618) [inline scripts/keep_sync.py:174]: we need to get if verticalAccuracy attr first.
if there is no value here
We do not  add it to the dict.

File: src/utils/utils.js (hunk 1 above)
Comment (abinnz) [reply src/utils/utils.js:37]: The length of the l variable may be equal to 1, so add the logic to judge the length.
```
const location = run.location_country;
  let [city, province, country] = ['', '', ''];
  if (location) {
    // Only for Chinese now
    const cityMatch = location.match(/[\u4e00-\u9fa5]*市/);
    const provinceMatch = location.match(/[\u4e00-\u9fa5]*省/);
    if (cityMatch) {
      [city] = cityMatch;
    }
    if (provinceMatch) {
      [province] = provinceMatch;
    }
    const l = location.split(',');
    // or to handle keep location format
    let countryMatch = l[l.length - 1].match(/[\u4e00-\u9fa5].*[\u4e00-\u9fa5]/);
    if (!countryMatch && l.length >= 3) {
      countryMatch = l[2].match(/[\u4e00-\u9fa5].*[\u4e00-\u9fa5]/);
    }
    if (countryMatch) {
      [country] = countryMatch;
    }
  }
```

File: scripts/keep_sync.py (hunk 2 above)
Comment (abinnz) [reply scripts/keep_sync.py:174]: Other props may not need to be judged to exist.
```
   # future to support heart rate
    points_dict_list = []
    for point in run_points_data:
        points_dict = {
            "latitude": point["latitude"],
            "longitude": point["longitude"],
            "time": datetime.utcfromtimestamp((point["timestamp"] * 100 + start_time) / 1000),
        }
        if "verticalAccuracy" in point.keys():
            points_dict["verticalAccuracy"] = point["verticalAccuracy"]
        points_dict_list.append(points_dict)
    gpx = gpxpy.gpx.GPX()
```

Comment (yihong0618) [reply scripts/keep_sync.py:174]: 1. only in is fine 
image
2. points_dict["elevation"] = point["verticalAccuracy"]

Comment (abinnz) [reply scripts/keep_sync.py:174]: Okay, I'll pr again.

**Comment by yihong0618** [comment]
Cool thanks a lot.
If you want you can share your page in https://github.com/yihong0618/running_page/issues/12

**Pull request merged**